README.md

```diff
@@ -275,40 +220,6 @@ http_client.headers.update({'x-custom-header': 'someValue'})
 s = shippo.Shippo(client: http_client)
 ```
 <!-- End Custom HTTP Client [http-client] -->
-
-<!-- Start Authentication [security] -->
-## Authentication
-
-### Per-Client Security Schemes
-
-This SDK supports the following security scheme globally:
-
-| Name             | Type             | Scheme           |
-| ---------------- | ---------------- | ---------------- |
-| `api_key_header` | apiKey           | API key          |
-
-To authenticate with the API the `api_key_header` parameter must be set when initializing the SDK client instance. For example:
-```python
-import shippo
-
-s = shippo.Shippo(
-    api_key_header="<YOUR_API_KEY_HERE>",
-)
-
-
-res = s.addresses.list_addresses(page=1, results=25, shippo_api_version='<value>')
-
-if res.address_list_wrapper is not None:
-    # handle response
-    pass
-
-```
-<!-- End Authentication [security] -->
-
-<!-- Placeholder for Future Speakeasy SDK Sections -->
```

Comment:
also, this might need to stay, though not really sure... the docs say that any future auto-generated sections will be added after this comment, not sure what the generator will do if it's not there (it may auto add it)

Reply:
re- added this